core/storage/collection/gae_models.py

```diff
@@ -229,6 +229,28 @@ def _trusted_commit(
                     self.status == constants.ACTIVITY_STATUS_PRIVATE)
             ).put_async()
 
+    @classmethod
+    def export_data(cls, user_id):
+        """(Takeout) Export user-relevant properties of CollectionRightsModel.
+        Args:
```

Comment:
Add blank line above this one, and also add a blank line above Returns.

Reply:
done